packages/openactive-integration-tests/test/shared-behaviours/common.js

```diff
@@ -54,6 +57,92 @@ class Common {
       });
     });
   }
+
+  /**
```

Comment:
These 2 functions are analogs of the old functions - but more generalized so they don't require `state` or `stage` (which no longer exist in FlowStage world)